src/sr_od/backend/backend_context.py

```
@@ -0,0 +1,837 @@
+            ``(ok, future)``：``ok=False`` 表示已有运行在进行(``future=None``)；
+            ``ok=True`` 表示已启动，``future`` 可供阻塞 await 取结果。
+        """
+        return self.run_slot._start(source, op_factory=op_factory, display_name=display_name)
+
+    def run_one_dragon(self, source: str) -> tuple[bool, Future | None]:
+        """按当前一条龙配置启动完整一条龙运行(app 路径,经 ``_start_app``)。"""
+        self._ensure_ready()
+        return self._start_app(source, application_const.ONE_DRAGON_APP_ID, application_const.DEFAULT_GROUP_ID)
+
+    def run_standalone_app(self, source: str, app_id: str | None = None) -> tuple[bool, Future | None]:
+        """启动独立应用；app_id 为空时使用 GUI 当前选中的独立应用(app 路径,经 ``_start_app``)。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)